draft-ietf-jsonpath-base.md

```diff
@@ -459,20 +455,17 @@ the internal workings of an implementation:  Implementations may wish
 conform to the model.
 
 In the processing model,
-a valid JSONPath query is executed against a Value, the *Argument*, and
-produces a list of zero or more nodes of the Value which are
-selected by the JSONPath.
-(Note that the term Value also implies that this value complies
-to the definitions in {{-json}}, i.e., is indeed a Value.)
+a valid query is executed against a value, the *argument*, and
+produces a list of zero or more nodes of the value.
 
-The JSONPath query is a sequence of zero or more *selectors*, each of
+The query is a sequence of zero or more *selectors*, each of
```

Comment:
We are still in the initial exposition, so redundancy is good here.

Reply:
The word "query" has no other usage within this spec other than to represent a JSONPath query.  The redundancy doesn't help clarify anything.  (It also appears I missed removing some self-references.)